my.css

```diff
@@ -0,0 +1,107 @@
+.main {
+	position: relative;
+	max-width: 1200px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5556249)